src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -31,7 +31,8 @@ RUN subscription-manager repos --enable codeready-builder-for-rhel-9-${ARCH}-rpm
     dnf install -y yum-utils gnupg git --allowerasing curl pkg-config clang perl libseccomp-devel gpgme-devel \
     device-mapper-devel qemu-kvm unzip wget libassuan-devel genisoimage cloud-utils-growpart cloud-init; 
 
-ADD --checksum=${YQ_CHECKSUM} https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_${ARCH/x86_64/amd64} /usr/local/bin/yq
+RUN curl -L -o /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_${ARCH/x86_64/amd64} \
```

Comment:
I believe this change is to make this work with podman. I suggest using a separate commit for this change

Reply:
Sure, ill create a separate commit for this change. Thank you